index.html

```diff
@@ -25,95 +25,333 @@
+                  <li>BootStrap</li>
+                  <li>Ruby on Rails</li>
+                  <li>RPSec</li>
+                  <li>CapyBase</li>
+                  <li>Selenium</li>
+                </ul>
+              </div>
+            </div>
+            <div>
+              <div class="technologies-item">
+                <div class="technologies-title">Skills</div>
+                <ul>
+                  <li>Codekit</li>
+                  <li>GitHub</li>
+                  <li>CodePen</li>
+                  <li>GitLab</li>
+                  <li>Terminal</li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <hr class="first-line" />
+
+          <!-- PAGE BREAK -->
+          <section class="page-break">
+            <div>
+              <p>Work &darr;</p>
+            </div>
+            <div class="social-links">
+              <ul>
+                <li>
+                  <a href="https://github.com/Marlyn_Mayienga"
+                    ><i class="fab fa-github"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                    ><i class="fab fa-linkedin"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.instagram.com/merl.__/?hl=en"
+                    ><i class="fas fa-cat"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://twitter.com/M_ayienga"
+                    ><i class="fab fa-twitter"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="marlymayienga@gmail.com"
+                    ><i class="fas fa-envelope"></i
+                  ></a>
+                </li>
+              </ul>
+            </div>
+            <div>
+              <p>&darr; Work</p>
+            </div>
+          </section>
         </div>
-        <div class="social-links">
+
+        <section class="my-recent-work">
+          <div class="container">
+            <h3>My Recent Work</h3>
+            <hr class="second-line" />
+          </div>
+
+          <div class="works">
+            <div class="work-item">
+              <img
+                class="ellipse-one"
+                src="./Ellipse.svg"
+                alt="project image"
+              />
+
+              <div class="work-item-content">
+                <img class="work-img" src="./project.jpg" alt="project image" />
+                <div class="work-content">
+                  <h4 class="work-item-title">Multi-Post Stories</h4>
+                  <p class="project-section">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
```

Comment:
- [ ] Kindly change the color for the paragraph to `#B3BAC5` to match the Figma design.